tests/drivers/build_all/rtc/testcase.yaml

```diff
@@ -11,3 +11,7 @@ tests:
       - arduino_gpio
     extra_args: SHIELD=adafruit_data_logger
     platform_allow: frdm_k64f
+  drivers.rtc.build.pcf8563:
```

Comment:
I now realize we need to do create a generic target to build the RTCs similar to what we have for sensors. Could you undo this addition in this PR? Sorry for the inconvenience.

TL:DR this addition will result in empty builds for all target platforms without including the driver, since there is no target specified, and AFAIK no board which includes the new RTC with node `status = "okay"`, this is what the `extra_args:` and `platform_allow:` options configure for the pcf8523

Reply:
it's no bother! I already reverted it. Please tell me if you found anything else